pages/ksphere/kommander/1.1.0-beta/operations/access-control/index.md

```diff
@@ -6,30 +6,47 @@ menuWeight: 1
 excerpt: Centrally manage access across clusters
 ---
 
-Role-based authorization can be defined centrally within Kommander to control resource access on all clusters or a subset of clusters. The resources are similar to Kubernetes RBAC but with crucial differences. These resouces make it possible to define the roles and policies once and have them federated to clusters within a given scope:
+Role-based authorization can be defined centrally within Kommander to control resource access on the management cluster, all target clusters, or a subset of target clusters. These resources are similar to Kubernetes RBAC but with crucial differences. These resources make it possible to define the roles and policies once and have them federated to clusters within a given scope.
 
-- Global or workspace-level roles and policies federate those resources to cluster-scoped roles and policies on all clusters. In the case of workspace-level resources, they only apply to workspace clusters.
+Kommander has two conceptual groups of resources that are used to manage access control:
+- Kommander roles: control access to resources on the management cluster.
+- Cluster Roles, Workspace Roles, and Project Roles: control access to resources on all target clusters in scope.
 
-- Project-level roles and policies federate namespace-scoped roles and policies to project clusters.
+These two groups of resources can be used to manage access control within 3 levels of scope:
+
+|           | Kommander Roles                                                                                         | Global/Workspace/Project Roles                                                                           |
+|-----------|---------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Global    | Global Kommander Roles create ClusterRoles on the management cluster.                                   | Global Cluster Roles federates ClusterRoles on all target clusters across all workspaces.                |
+| Workspace | Workspace Kommander Roles create namespaced Roles on the management cluster in the workspace namespace. | Workspace Cluster Roles federates ClusterRoles on all target clusters in the workspace.                  |
```

Comment:
Workspace Kommander Roles also create roles on the management cluster in the workspace project namespaces.

Reply:
@natmegs